src/Provider/LineItemImagesProvider.php

```diff
@@ -60,7 +65,16 @@ public function getImageUrlFromProduct(ProductInterface $product): string
 
     private function getUrlFromPath(string $path): string
     {
-        $url = $this->filterExtension->filter($path, $this->filterName);
+        // if given path is empty, InvalidParameterException will be thrown in filter action
+        if (empty($path)) {
+            return $this->fallbackImage;
+        }
+
+        try {
+            $url = $this->filterExtension->filter($path, $this->filterName);
+        } catch (\Exception $e) {
```

Comment:
Catching exception here will avoid error in production indeed, but you will never know that the problem will come from here and you won't be able to test it in you dev env too.
What do you think about removing the try catch ? Because you already avoid a specific error, each futur one have to be spotted if they are related to this Plugin.

Reply:
If exception happen, fallback image will be displayed in stripe checkout.
When users will see that, they will know there is a problem, so, we will warned.

I prefer don't block the production checkout process just for an image problem.

Now, the choice is yours... 😉

Reply:
I agree the process should not be stopped in this case 👍